kie-server-parent/kie-server-controller/kie-server-controller-rest/src/main/java/org/kie/server/controller/rest/RestSpecManagementServiceImpl.java

```diff
@@ -108,10 +108,6 @@ public Response saveServerTemplate(@Context HttpHeaders headers, @PathParam("id"
 
         String contentType = getContentType(headers);
         try {
-            if (super.getServerTemplate(serverTemplateId) != null) {
```

Comment:
@mswiderski in this case specifically I'm changing the REST API behavior to match the base service, where the server template is updated instead of error when a server with the same id already exists.

Reply:
@cristianonicolai fine with me